addons/configtweaks/config.cpp

```diff
@@ -174,14 +183,13 @@ class CfgVehicles {
         explosionShielding = 1;
         visual = "";
         minimalHit = 0;
-        depends =
-            "(((Total - 0.25) max 0) + ((HitHead - 0.25) max 0)"
-            "+ ((HitBody - 0.25) max 0)) * 2";
+	depends="""(((Total - 0.25) max 0) + ((HitHead - 0.25) max 0)""" \n """+ ((HitBody - 0.25) max 0)) * 2""";
```

Comment:
Up to you how you want to format this, but I'd say that this isn't the ideal state 😂

Reply:
It's how it was originally, at least in the old configTweaks and I guess BRH.

Reply:
Fixed to actual original BRH line now